ios/Exponent/Kernel/Services/Notifications/EXApiV2Client+EXRemoteNotifications.h

```diff
@@ -9,6 +9,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (NSURLSessionTask *)updateDeviceToken:(NSData *)deviceToken
                       completionHandler:(void (^)(NSError * _Nullable postError))handler;
 - (NSURLSessionTask *)getExpoPushTokenForExperience:(NSString *)experienceStableLegacyId
+                                       easProjectId:(NSString *)easProjectId
```

Comment:
Since the EAS project ID is the "main ID" moving forward, it might make sense to rename this method to `expoPushTokenForEASProject:experience:...`

Also, both ID parameters now need to be marked as `nullable` AFAICT

Reply:
ah, good catch. Forgot this was in an `NS_ASSUME_NONNULL_BEGIN`.